db/schema.rb

```diff
@@ -282,9 +282,6 @@
     t.index ["uuid"], name: "index_courses_on_uuid", unique: true
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
Was this meant to be removed :thinking:

Reply:
I ran `rails db:drop db:migrate` and it updated the schema. Maybe the migration got removed for that table? 🤔

Reply:
Yeah, pretty sure we don't want to remove this 💣